web/src/utils/env.ts

```diff
@@ -27,6 +27,7 @@ if (process.env.NODE_ENV === 'production') {
     KC_AUTH_URL: window.env.REACT_APP_KEYCLOAK_AUTH_URL,
     KC_REALM: window.env.REACT_APP_KEYCLOAK_REALM,
     KC_CLIENT: window.env.REACT_APP_KEYCLOAK_CLIENT,
+    WF1_AUTH_URL: window.env.REACT_APP_WF1_AUTH_URL,
```

Comment:
```suggestion
```
I don't think we need `WF1_AUTH_URL` anymore

Reply:
done